### DISCUSSION TITLE
Unable to get translated text

### DISCUSSION THREAD

**Discussion by kaisiant** [vinissimus/next-translate#443] [Q&A]
This is my locales file en/common.json
```
{
   "strategy.research": "Strategy Research",
}
```

I try to get it with: 
```
const { t } = useTranslation('common')
const test = t('strategy.research') // return "common:strategy.research"

```

Is it we should using this "-" instead of "." as "." is for nested obj?

**Comment by aralroca** [comment]
The dot `.` is a reserved character for nesting objects:

```json
{
   "strategy": {
      "research": "Strategy Research",
   }
}
```

```js
const test = t('strategy.research')
```

Code: https://github.com/vinissimus/next-translate/blob/master/src/transCore.tsx#L60

Would be great to do a fix to allow also `.` inside the key. Feel free to do PR to fix this.
